src/main/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsController.java

```diff
@@ -11,41 +11,31 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
-import uk.gov.hmcts.reform.sscs.exception.AuthorisationException;
 import uk.gov.hmcts.reform.sscs.exception.GetCaseException;
-import uk.gov.hmcts.reform.sscs.exception.InvalidHeaderException;
 import uk.gov.hmcts.reform.sscs.exception.InvalidIdException;
 import uk.gov.hmcts.reform.sscs.helper.mapping.LinkedCasesMapping;
 import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.ServiceHearingValues;
 import uk.gov.hmcts.reform.sscs.model.service.linkedcases.LinkedCase;
 import uk.gov.hmcts.reform.sscs.model.service.linkedcases.ServiceLinkedCases;
-import uk.gov.hmcts.reform.sscs.service.AuthorisationService;
 import uk.gov.hmcts.reform.sscs.service.CcdCaseService;
 
 import java.util.List;
 
-import static io.swagger.v3.oas.annotations.enums.ParameterIn.HEADER;
 import static io.swagger.v3.oas.annotations.enums.ParameterIn.QUERY;
 import static org.springframework.http.ResponseEntity.status;
-import static uk.gov.hmcts.reform.sscs.service.AuthorisationService.SERVICE_AUTHORISATION_HEADER;
 
 @RestController
 @Slf4j
 public class ServiceHearingsController {
 
-    private final AuthorisationService authorisationService;
     private final CcdCaseService ccdCaseService;
 
     @Autowired
```

Comment:
I'm surprised we need to explicitly specify @Autowired here given it's a RestController.  Would have expected that to have been a given!  Can we try without?

Reply:
We should be able to, yes

Reply:
@nickwest1 A lot of this codebase has random @Autowired everywhere.. that's not been needed since Spring Boot v1 or older Spring code.. since it's not relevant to this PR I wouldn't waste effort on it.

Reply:
I wholly admit the @Autowired thing tripped me up! I'm used to xml config more